src/Common.jl

```diff
@@ -32,7 +32,8 @@ function G_func(param_set::APS, T::FT, ::TD.Liquid) where {FT <: Real}
     _R_v::FT = CP_planet.R_v(param_set)
     _D_vapor::FT = CP_micro.D_vapor(param_set)
 
-    L = TD.latent_heat_vapor(param_set, T)
+    L = CP_planet.LH_v0(param_set)
```

Comment:
```suggestion
    L::FT = CP_planet.LH_v0(param_set)
```
Previously the type could be promoted from `T`, now we need to type-cast for type stability